recipes/uv-dynamic-versioning/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "uv-dynamic-versioning" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/uv_dynamic_versioning-{{ version }}.tar.gz
+  sha256: 2e0e4f6fbd9ae4507a9f5bbbc2c55266afce1042ee9c77358c1e4c1ae6453a1e
+
+build:
+  skip: true  # [py>=400 or py2k]
```

Comment:
This `skip` should be removed and this recipe converted to `noarch: python`:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python

Reply:
Thanks for reviewing and let me know if [3f7acab](https://github.com/conda-forge/staged-recipes/pull/29005/commits/3f7acab3b7dd1ba75fdf15ebebbbbe7aaf31ec81) is not enough.